test/DurableTask.SqlServer.AzureFunctions.Tests/CoreScenarios.cs

```diff
@@ -154,7 +154,155 @@ public async Task MultiInstanceQueries()
+            Assert.All(result.DurableOrchestrationState, state => Assert.Equal(OrchestrationRuntimeStatus.Completed, state.RuntimeStatus));
+
+            statusFilters.Clear();
+            result = await client.ListInstancesAsync(condition, CancellationToken.None);
+            Assert.Equal(10, result.DurableOrchestrationState.Count());
+
+            // Test InstanceIdPrefix
+            condition.InstanceIdPrefix = "Foo";
+            result = await client.ListInstancesAsync(condition, CancellationToken.None);
+            Assert.Empty(result.DurableOrchestrationState);
+
+            condition.InstanceIdPrefix = prefix;
+            result = await client.ListInstancesAsync(condition, CancellationToken.None);
+            Assert.Equal(10, result.DurableOrchestrationState.Count());
+
+            // Test PageSize and ContinuationToken
+            var instanceIds = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
+            condition.PageSize = 0;
+            while (condition.PageSize++ < 10)
+            {
+                result = await client.ListInstancesAsync(condition, CancellationToken.None);
+                int total = result.DurableOrchestrationState.Count();
+                Assert.Equal(condition.PageSize, total);
+
+                // Make sure all instance IDs are unique
+                Assert.All(result.DurableOrchestrationState, s => instanceIds.Add(s.InstanceId));
+
+                while (total < 10)
+                {
+                    condition.ContinuationToken = result.ContinuationToken;
+                    result = await client.ListInstancesAsync(condition, CancellationToken.None);
+                    int count = result.DurableOrchestrationState.Count();
+                    Assert.NotEqual(0, count);
+                    Assert.True(count <= condition.PageSize);
+                    total += count;
+                    Assert.True(total <= 10);
+                    Assert.All(result.DurableOrchestrationState, s => instanceIds.Add(s.InstanceId));
+                }
+
+                condition.ContinuationToken = null;
+            }
+
+            // Test ShowInput
+            condition.ShowInput = true;
+            condition.CreatedTimeFrom = sequencesFinishedTime;
+            result = await client.ListInstancesAsync(condition, CancellationToken.None);
+            Assert.All(result.DurableOrchestrationState, state => Assert.NotEqual(JValue.CreateNull(), state.Input));
+
+            condition.ShowInput = false;
+            condition.CreatedTimeFrom = startTime;
+            result = await client.ListInstancesAsync(condition, CancellationToken.None);
+            Assert.All(result.DurableOrchestrationState, state => Assert.Equal(JValue.CreateNull(), state.Input));
+        }
+
+         [Fact]
+        public async Task MultiInstanceQueriesWithoutMultiTenancy()
+        {
+            await SharedTestHelpers.DisableMultitenancyAsync();
+            await SharedTestHelpers.PurgeAsync();
```

Comment:
Is the only difference between these two tests the disabling of multitenancy? Can you clarify what issues this variant catches that the existing test doesn't?

Reply:
I added 2 more clear tests on the difference between  with and without